front/lib/models/assistant/actions/tables_query.ts

```diff
@@ -130,13 +130,13 @@ AgentTablesQueryConfigurationTable.init(
     indexes: [
       {
         unique: true,
-        fields: [
-          "dataSourceWorkspaceId",
-          "dataSourceId",
-          "tableId",
-          "tablesQueryConfigurationId",
-        ],
-        name: "agent_tables_query_configuration_table_unique",
+        fields: ["dataSourceViewId", "tableId", "tablesQueryConfigurationId"],
+        name: "agent_tables_query_configuration_table_unique_dsv",
+      },
+      {
+        unique: true,
```

Comment:
This is not correct as we can technically have multiple tables from multiple views on the same data source for the same table query configuration (the product allows that). That flow would error due to the unique:true here.